en/docs/learn/metadata-cache-management.md

```diff
@@ -0,0 +1,144 @@
+Data Holders are required to be able to react to Data Recipient and associated software product status changes within
+2-5 minutes of the change occurring on the CDR Register. To achieve this, Data Holders can poll APIs to retrieve the
+current statuses and check the validity of Data Recipients and associated software products and cache these for use
+during requests for consumer data.
+
+Metadata Cache Management module of WSO2 Open Banking supports:
+
+1. Retrieving the statuses of Accredited Data Recipient (ADR) software products from the Australian Competition and
+   Consumer Commission (ACCC).
+
+2. Validating ADR and software product during:
+
+- Requests to disclose CDR data
+- Requests to withdraw CDR Arrangement ID
+- Consumer authorizes the CDR Arrangement ID
+
+3. Performing Data Holder responsibilities such as revoking CDR Arrangement ID and removing client registrations.
```

Comment:
check the formatting of this point as well. 
<img width="718" alt="Screenshot 2022-04-21 at 14 24 34" src="https://user-images.githubusercontent.com/14251853/164418717-46a2267c-b313-4982-93b0-4e5d6205d711.png">